proxy/src/main/java/com/velocitypowered/proxy/config/VelocityConfiguration.java

```diff
@@ -94,6 +94,18 @@
   @Ignore
   private @Nullable Favicon favicon;
 
+  @Comment({"What TCP Fast Open mode should be used? (Works only if Epoll is available)",
+          "0 - disabled", "1 - TFO is only enabled for outgoing client",
+          "2 - TFO is only available on listening server",
+          "3 - includes 1 and 2"})
+  @ConfigKey("tcp-fast-open")
+  private int tcpFastOpenMode = 0;
```

Comment:
Make this an enum?

Reply:
Yes, this definitely needs to be an enum.